source/coll_common.f90

```diff
@@ -63,6 +63,7 @@ module coll_common
   character(len=11), parameter :: coll_tracksFile     = "tracks2.dat"
   character(len=17), parameter :: coll_positionsFile  = "CollPositions.dat"
   character(len=20), parameter :: coll_pencilFile     = "pencilbeam_distr.dat"
+  character(len=23), parameter :: coll_pencilFileRB   = "pencilbeam_distr_RB.dat"
```

Comment:
Why "RB"?

Reply:
I assume this is because roderik intially wrote this bit of code

Reply:
Yeah, not a good naming convention. I suggested renaming it to `pencilbeam_distr_3.dat` instead.

Reply:
ok done